pyaedt/modules/MaterialLib.py

```diff
@@ -870,3 +870,19 @@ def import_materials_from_excel(self, material_file):
             materials_added.append(new_material)
 
         return materials_added
+
+    @pyaedt_function_handler
+    def get_used_project_materials(self):
+        """Get list of material names in current design.
```

Comment:
Few comments:
- does it returns the materials used in the design or in the project?
- this function should return a list of material `Material` objects.
- if you want you can also create a class which returns the list of material names (`get_used_project_material_names`)

Reply:
@Alberto-DM @Samuelopez-ansys 
- This method return a list or material names used in project. I could rename it as `get_design_materials` and change the logic. In this case, can we keep this method in `MaterialLib`?
- why material objects rather than material names?

Reply:
@Alberto-DM @Samuelopez-ansys for me it's still not clear how to retrieve only materials used in a specific design. The `is_used` prop doesn't specify if the material is used in my design (in case of multiple designs in a project).

Reply:
@Alberto-DM @gmalinve What this method is doing is getting the material names used in the project, so @Alberto-DM suggestion is correct, and change the name of the method to "get_used_project_material_names" because it is consistent to other pyaedt methods.

I think we can merge this pull request because it is a useful method, but this will follow in the next implementation, to get the materials used in a design, because this is more useful than getting the project materials.